src/MPDevice.cpp

```diff
@@ -1994,7 +1994,7 @@ bool MPDevice::addChildToDB(MPNode* parentNodePt, MPNode* childNodePt)
         else
         {
             /* If the login name is supposed to be before the one we're trying to insert */
-            if (tempNextChildNodePt->getLogin() > childNodePt->getLogin())
+            if (tempNextChildNodePt->getLogin().compare(childNodePt->getLogin()) > 0)
```

Comment:
Maybe better using QCollator here for comparing strings?
http://doc.qt.io/qt-5/qcollator.html
https://wiki.qt.io/QCollator